infrastructure-playbooks/filestore-to-bluestore.yml

```diff
@@ -39,17 +39,18 @@
       block:
         - import_role:
             name: ceph-facts
+            tasks_from: container_binary.yml
+
+        - name: set_fact container_run_cmd
+          set_fact:
+            container_run_cmd: "{{ container_binary + ' run --rm --privileged=true --net=host --pid=host --ipc=host -v /dev:/dev -v /etc/ceph:/etc/ceph -v /var/lib/ceph:/var/lib/ceph -v /var/run:/var/run --entrypoint=' if containerized_deployment else '' }}ceph-volume {{ ceph_docker_registry + '/' + ceph_docker_image + ':' + ceph_docker_image_tag if containerized_deployment else '' }}"
 
         - name: get ceph osd tree data
           command: "{{ container_exec_cmd }} ceph osd tree -f json"
```

Comment:
Since we only import `container_binary` tasks then I think `container_exec_cmd` should be replace by `container_run_cmd` right ?

Reply:
in fact container_run_cmd fact is for ceph-volume commands whereas we need something almost the same for ceph commands

Reply:
tru.